src/file-uploader/file-uploader.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface FileItem {
+	file: File;
+	state: string;
+	uploaded: boolean;
```

Comment:
I wonder if there's a way to automatically propagate the `uploaded` prop 🤔 we don't have to worry about it now, but it'd be cool if we could handle that somehow